doc/src/manual/faq.md

```diff
@@ -593,6 +593,12 @@ julia> remotecall_fetch(anon_bar, 2)
 1
 ```
 
+### Why does Julia use "*" for string concatenation? Why not "+" or something else?
```

Comment:
You can use code formatting in headings, so maybe update `"*"` -> `` `*` `` and likewise for `+`?